wire-ios/Wire-iOS/Sources/UserInterface/StartUI/Common/Cells/OpenServicesAdminCell.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+final class OpenServicesAdminCell: StartUIIconCell {
+
+    override func setupViews() {
+        super.setupViews()
+        icon = .bot
+        title = PeoplePicker.QuickAction.adminServices
+        isAccessibilityElement = true
+        accessibilityLabel = title
+        accessibilityIdentifier = "button.searchui.open-services"
```

Comment:
Do we also have generated access like `L10n` for accessibility?

Reply:
Yes, But I would like to focus on accessibility on a separate and dedicated PR. It's on my list.